src/lib/server/services/content.spec.ts

```diff
@@ -91,6 +93,61 @@ const generateStatContext = async (db: DB) => {
     };
 };
 
+
+const generateComments = async (db: DB) => {
+    const [creator] = await db.insert(userTable).values({ username: 'AwesomeGuy' }).returning();
+    const [creator2] = await db.insert(userTable).values({ username: 'AwesomeGuy1' }).returning();
+    const [creator3] = await db.insert(userTable).values({ username: 'AwesomeGuy2' }).returning();
+    const [creator4] = await db.insert(userTable).values({ username: 'AwesomeGuy3' }).returning();
+
+    const [channel] = await db
+        .insert(channelTable)
+        .values({ name: 'Channel-Surfers', createdBy: creator.id })
+        .returning();
+
+    const [post1] = await db
+        .insert(postTable)
+        .values([
+            { channelId: channel.id, createdBy: creator.id, title: 'Awesome post 1', videoId: '' }
+        ])
+        .returning();
+
+    const January1 = new Date(); January1.setMonth(1); January1.setDate(1);
+    const January2 = new Date(); January2.setMonth(1); January2.setDate(2);
+    const January3 = new Date(); January3.setMonth(1); January3.setDate(3);
+    const January4 = new Date(); January4.setMonth(1); January4.setDate(4);
```

Comment:
Use the `Date` constructor to set the year, month, and day: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#syntax